### ISSUE TITLE
Removing Causal Component from WAN VAE

### ISSUE THREAD

**Issue by ktertikas** [imlixinyang/FlashWorld#21]
Hey @imlixinyang ,

It seems like in the code you remove the padding from the WAN VAE that corresponds to the temporal component, effectively removing the Causality component here:

https://github.com/imlixinyang/FlashWorld/blob/b4857e3a66ff925cbd9cae09286f9de9ea73c781/app.py#L152-L161

From an information propagation perspective that makes sense, as it forces the latents to have information from all the input frames, and this would be useful when we want to generate a 3D scene out of all input frames. However, it seems a bit strange to me that this works out of the box, as there is no finetuning in the VAE. Could you comment about this part, and explain your intuition behind this? 

Best,
Konstantinos

**Comment by imlixinyang** [comment]
Yes, all frames are encoded independently during pre-training and post-training, even for video data. However, compared to the original sampling rate of approximately 15 to 30 FPS for wan, we sample frames with larger intervals (5 FPS to 1 FPS).

Intuitively, 3D generation does not rely much on continues frames. Using keyframes can achieve a larger scene scale under similar latent dimensions.
